modules/integration/tests-integration/tests-es/src/test/java/org/wso2/carbon/registry/es/notes/GregWsdlNotesESTestCase.java

```diff
@@ -0,0 +1,206 @@
+        //  https://localhost:9443/publisher/apis/assets?type=note
+        ClientResponse response = genericRestClient
+                .geneticRestRequestDelete(publisherUrl + "/assets/" + noteAssetId, MediaType.APPLICATION_JSON,
+                        MediaType.APPLICATION_JSON, queryParamMap, headerMap, cookieHeaderPublisher);
+
+        response.getEntity(String.class).contains("Asset Deleted Successfully");
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void cleanUp() throws RegistryException {
+        deleteAsset(assetId, publisherUrl, cookieHeaderPublisher, assetType, genericRestClient);
+    }
+
+    private void SetTestEnvironment() throws JSONException, XPathExpressionException,
+            IOException {
+        JSONObject objSessionPublisher = new JSONObject(authenticate(publisherUrl, genericRestClient,
+                automationContext.getSuperTenant().getTenantAdmin().getUserName(),
+                automationContext.getSuperTenant().getTenantAdmin().getPassword()).getEntity(String.class));
+        jSessionId = objSessionPublisher.getJSONObject("data").getString("sessionId");
+        cookieHeaderPublisher = "JSESSIONID=" + jSessionId;
+        esTestCommonUtils.setCookieHeader(cookieHeaderPublisher);
+
+        String dataBody = readFile(resourcePath + "wsdl-ops.json");
+        assetName = (String)(new JSONObject(dataBody)).get("overview_name");
+        JSONObject objSessionId = new JSONObject(
+                createAsset(resourcePath + "wsdl-ops.json", publisherUrl, cookieHeaderPublisher, assetType,
+                        genericRestClient).getEntity(String.class));
+
+        String resultName = objSessionId.get("overview_name").toString();
+        Assert.assertEquals(resultName,assetName);
+        searchWsdlAsset();
+    }
+
+    public void searchWsdlAsset() throws JSONException {
+        boolean assetFound = false;
+        Map<String, String> queryParamMap = new HashMap<String, String>();
+        queryParamMap.put("type", "wsdl");
+        queryParamMap.put("overview_name", assetName);
+        ClientResponse clientResponse = esTestCommonUtils.searchAssetByQuery(queryParamMap);
+        JSONObject obj = new JSONObject(clientResponse.getEntity(String.class));
+        JSONArray jsonArray = obj.getJSONArray("list");
+        for (int i = 0; i < jsonArray.length(); i++) {
+            String name = (String)jsonArray.getJSONObject(i).get("name");
+            if (assetName.equals(name)) {
+                assetFound = true;
+                assetId = (String)jsonArray.getJSONObject(i).get("id");
+                break;
+            }
+        }
+        Assert.assertEquals(assetFound,true);
+        Assert.assertNotNull(assetId, "Empty asset resource id available");
+    }
+
+    @DataProvider
+    private static TestUserMode[][] userModeProvider() {
+        return new TestUserMode[][]{
+                new TestUserMode[]{TestUserMode.SUPER_TENANT_ADMIN},
+        };
+    }
+}
```

Comment:
Thanks. I have reformatted with the following commit - https://github.com/wso2/product-greg/pull/274/commits/470633fc39549bf01b238abf015cd6d5abdc11bb